clients/src/test/java/org/apache/kafka/clients/admin/internals/DeleteConsumerGroupOffsetsHandlerTest.java

```diff
@@ -149,4 +191,21 @@ private void assertFailed(
         assertEquals(singleton(key), result.failedKeys.keySet());
         assertTrue(expectedExceptionType.isInstance(result.failedKeys.get(key)));
     }
-}
+
+    private void assertPartitionFailed(
+        Map<TopicPartition, Errors> expectedResult,
+        AdminApiHandler.ApiResult<CoordinatorKey, Map<TopicPartition, Errors>> result
+    ) {
+        CoordinatorKey key = CoordinatorKey.byGroupId(groupId);
+        assertEquals(singleton(key), result.completedKeys.keySet());
+
+        // verify the completed value is expected result
+        Collection<Map<TopicPartition, Errors>> completeCollection = result.completedKeys.values();
+        assertEquals(1, completeCollection.size());
+        Map<TopicPartition, Errors> completeMap = completeCollection.iterator().next();
+        assertEquals(expectedResult, completeMap);
```

Comment:
You already assert that `completedKeys` only contains `key` so it seems that we could just verify that `result.completedKeys.get(key)` is equal to `expectedResult`, no?

Reply:
Good suggestion! Updated.